cmd/yorkie/user/delete-account.go

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2024 The Yorkie Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package user
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"golang.org/x/term"
+
+	"github.com/yorkie-team/yorkie/admin"
+	"github.com/yorkie-team/yorkie/cmd/yorkie/config"
+)
+
+func deleteAccountCmd() *cobra.Command {
+	return &cobra.Command{
+		Use:     "delete-account",
+		Short:   "Delete user account",
+		PreRunE: config.Preload,
+		RunE: func(_ *cobra.Command, args []string) error {
+			password, err := getPassword()
+			if err != nil {
+				return err
+			}
+
+			if confirmation, err := makeConfirmation(); !confirmation || err != nil {
+				if err != nil {
+					return err
+				}
+				return nil
+			}
+
+			conf, err := config.Load()
+			if err != nil {
+				return err
+			}
+
+			if rpcAddr == "" {
+				rpcAddr = viper.GetString("rpcAddr")
+			}
+
+			if err := deleteAccountFromServer(conf, rpcAddr, insecure, username, password); err != nil {
+				fmt.Println("Failed to delete your account." +
+					"The account may not exist or the password might be incorrect. Please try again.")
+			} else {
+				fmt.Println("Your account has been successfully deleted.")
+			}
+
+			return nil
+		},
+	}
+}
+
+func getPassword() (string, error) {
+	fmt.Print("Enter Password: ")
+	bytePassword, err := term.ReadPassword(int(os.Stdin.Fd()))
+	if err != nil {
+		return "", fmt.Errorf("failed to read password: %w", err)
+	}
+	password = string(bytePassword)
+	fmt.Println()
+	return password, nil
```

Comment:
One last comment: spacing before return :)

Reply:
I'm sorry to bother you about the format.
Thank you for your detailed review..!